corehq/toggles.py

```diff
@@ -1638,7 +1638,8 @@ def _commtrackify(domain_name, toggle_is_enabled):
 
 FILTER_ON_GROUPS_AND_LOCATIONS = StaticToggle(
     'filter_on_groups_and_locations',
-    'Filter on groups AND locations in all reports with group and location filters',
+    '[ONSE] Toggle filter from groups OR locations to groups AND locations in all reports and exports in the '
```

Comment:
I think the phrase toggle makes this more confusing. to me it implies that enables toggling this logic, but I think that this feature flag changes the logic to AND instead of OR without the ability to toggle back and forth